From c0c0375245d91056b8d660bc063baad538555f86 Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Mon, 20 May 2019 16:17:34 +0800
Subject: [PATCH 1488/1552] media: intel-ipu4: ox03a10: change output fps to 30

change output fps to 30

Change-Id: I90ff27597474d773caac34f657170a6ab9eea7ee
Tracked-On: PKT-2588
Tracked-On: #JIIAP-797
Signed-off-by: qianmenx <qianx.meng@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index 1765cb2..636ac23 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -253,8 +253,6 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x3809, CRL_REG_LEN_08BIT, 0x80 },
 	{ 0x380a, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x380b, CRL_REG_LEN_08BIT, 0x40 },
-	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
-	{ 0x380f, CRL_REG_LEN_08BIT, 0x40 },
 	{ 0x3810, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x3813, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x381c, CRL_REG_LEN_08BIT, 0x08 },
@@ -522,7 +520,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x50c2, CRL_REG_LEN_08BIT, 0xb0 },
 	{ 0x5102, CRL_REG_LEN_08BIT, 0xb0 },
 	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
-	{ 0x380f, CRL_REG_LEN_08BIT, 0x34 },
+	{ 0x380f, CRL_REG_LEN_08BIT, 0x6d },
 	{ 0x380c, CRL_REG_LEN_08BIT, 0x08 },
 	{ 0x380d, CRL_REG_LEN_08BIT, 0x78 },
 	{ 0x384c, CRL_REG_LEN_08BIT, 0x02 },
-- 
2.7.4

